[2021-10-02 18:59:10] [WARNING]: lambda 初始化失败: Post "http:///runtime/init/ready": http: no Host in request URL 
[2021-10-02 18:59:10] [WARNING]: 警告: pprof 服务不支持鉴权, 请不要运行在公网. 
[2021-10-02 18:59:12] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:17] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:22] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:27] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:32] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:37] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:42] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:47] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:52] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 18:59:57] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:00:02] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:00:07] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:00:13] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:00:18] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:00:23] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:00:28] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:00:33] [WARNING]: 连接到反向WebSocket Universal服务器 ws://127.0.0.1:8080/cqhttp/ws 时出现错误: dial tcp 127.0.0.1:8080: connectex: No connection could be made because the target machine actively refused it. 
[2021-10-02 19:01:58] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-02 19:02:02] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-02 19:33:49] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:59893->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-02 19:33:50] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:59893->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
[2021-10-02 19:33:50] [ERROR]: Protocol -> sendPacket msg error: 120  
[2021-10-02 19:33:55] [WARNING]: 群消息发送失败: 账号可能被风控. 
[2021-10-02 20:20:04] [WARNING]: 监听反向WS API时出现错误: read tcp 127.0.0.1:53723->127.0.0.1:8080: wsarecv: An existing connection was forcibly closed by the remote host. 
[2021-10-02 20:20:05] [WARNING]: 向WS服务器 127.0.0.1:8080 推送Event时出现错误: write tcp 127.0.0.1:53723->127.0.0.1:8080: wsasend: An existing connection was forcibly closed by the remote host. 
